pkg/sharer/sharer.go

```diff
@@ -168,7 +168,7 @@ func NewSQLSharer(logger log.Logger, driver string, dsn string, createTables boo
 			}
 		}
 
-	case "sqlite3":
+	case "sqlite":
```

Comment:
@juliusv What do you think of this:

```suggestion
	case "sqlite3", "sqlite":
```

Reply:
Yeah, let's do that, great!

Reply:
Done, took a slightly different tactic, but will also nicely log a warning now.